### ISSUE TITLE
xor code 작성

### ISSUE THREAD

**Issue by yuseungwoo** [yuseungwoo/pytorch#4]
import torch as tc
import torch.nn as nn

tc.manual_seed(70)

class Network(nn.Module):
    def __init__(self, input_size, hidden_size, output_size):
        super(Network, self).__init__()
        self.ih = nn.Linear(input_size, hidden_size)
        self.ho = nn.Linear(hidden_size, output_size)
        self.act = nn.Sigmoid()
    
    def identity_func(self, x):
        return x
    
    def forward(self, x):
        z1 = self.ih(x)
        a1 = self.act(z1)
        z2 = self.ho(a1)
        y = self.identity_func(z2)
        return y

class Network2(nn.Module):
    def __init__(self,i_size,h_size,o_size):
        super(Network2,self).__init__()
        self.ih=nn.Linear(i_size,h_size)
        self.ho=nn.Linear(h_size,o_size)
        self.sig=nn.Sigmoid()
    
    def identy(self,x):
        return x
    
    def forward(self,x):
        z1=self.ih(x)
        a1=self.sig(z1)
        z2=self.ho(a1)
        a2=self.identy(z2)
        
        return a2
        

x = tc.Tensor([[0,1]])
net=Network2(2,2,1)
y=net(x).item()
print(y)
